AutoRest/Generators/Azure.Common/Azure.Common/AzureCodeGenerator.cs

```diff
@@ -454,6 +459,65 @@ public static void AddPageableMethod(ServiceClient serviceClient)
             }
         }
 
+        public static string GetClientRequestIdString(Method method)
+        {
+            if (method == null)
+            {
+                throw new ArgumentNullException("method");
+            }
+            
+            string clientRequestIdName = null;
```

Comment:
can you set the default value of ""x-ms-client-request-id"" here? This way you don't need the "if" block later